src/Xhgui/Searcher/Pdo.php

```diff
@@ -146,10 +146,28 @@ public function getAll($options = array())
     {
         $sort = $options['sort'];
         $direction = $options['direction'];
-        $page = $options['page'];
-        $perPage = $options['perPage'];
+        $page = (int)$options['page'];
+        if ($page < 1) {
+            $page = 1;
+        }
+        $perPage = (int)$options['perPage'];
+        $url = $options['conditions']['url'] ?? "";
```

Comment:
this syntax is not supported by php 5.6, which is this project min requirement:
- https://github.com/perftools/xhgui/blob/0.11.0/composer.json#L11